torch/_torch_docs.py

```diff
@@ -152,6 +152,29 @@ def merge_dicts(*dicts):
     tensor([ 1.2294,  2.2004,  1.3690,  1.7298])
 """.format(**common_args))
 
+add_docstr(torch.acosh,
+           r"""
+acosh(input, out=None) -> Tensor
+
+Returns a new tensor with the inverse hyperbolic cosine of the elements of :attr:`input`,
+All values in the input tensor must be greater than or equal to one.
+
+.. math::
+    \text{out}_{i} = \cosh^{-1}(\text{input}_{i})
+""" + r"""
+Args:
+    input (Tensor): the input tensor where :math:`input` is in the closed interval `[1, +inf)`.
```

Comment:
See above

Reply:
Done! Please check the doc language, in case you suggest it to be made simpler.